packages/opentelemetry-core/test/trace/ProbabilitySampler.test.ts

```diff
@@ -36,7 +36,7 @@ describe('ProbabilitySampler', () => {
   it('should return a always sampler for >1', () => {
     const sampler = new ProbabilitySampler(100);
     assert.strictEqual(sampler.shouldSample(), true);
-    assert.strictEqual(sampler.toString(), 'AlwaysSampleSampler');
+    assert.strictEqual(sampler.toString(), 'ProbabilitySampler{100}');
```

Comment:
This should be `ProbabilitySampler{1}` since the value should be capped.

Reply:
Sure, done,